docs/source/release/v6.2.0/indirect_geometry.rst

```diff
@@ -5,43 +5,47 @@ Indirect Geometry Changes
 .. contents:: Table of Contents
    :local:
 
+New Features
+------------
+- Fit functions :ref:`ElasticIsoRotDiff <func-ElasticIsoRotDiff>` and :ref:`InelasticIsoRotDiff <func-InelasticIsoRotDiff>` have been made available in the :ref:`ConvFit <convfit>` tab in the :ref:`Inelastic Data Analysis <interface-inelastic-data-analysis>` interface.
+- - **The** :ref:`Elwin <Elwin-iqt-ref>` **tab in** :ref:`Inelastic Data Analysis <interface-inelastic-data-analysis>` **has a new loader which now allows users to add workspaces.**
```

Comment:
This shouldn't be bold - we tend to put all release notes in at the same emphasis with exceptions for very big ticket items (Python 2 -> 3).